index.html

```diff
@@ -247,36 +247,38 @@ <h2>Let's Connect</h2>
     <div class="languages-learned">
       <ul class="languages">
          <li class="language-item">
-            <h1>Languages</h1>
+            <h3>Languages</h3>
               <a href="#"><i class="fa-solid fa-angle-down"></i></a>
          </li>
+         </ul>
          <div class="language-body">
             <div class="language">
               <div class="language-icon">
-                <img src="./Images/Ellipse 1.png" alt="">
+                <img src="./Images/Ellipse-js.png" alt="JavaScript icon">
               </div>
               <span>JavaScript</span>
             </div>
             <div class="language">
               <div class="language-icon">
-                <img src="./Images/Ellipse_2.png" alt="">
+                <img src="./Images/Ellipse_2.png" alt="HTML5 icon">
               </div>
               <span>HTML</span>
             </div>
             <div class="language">
               <div class="language-icon">
-                <img src="./Images/Ellipse_3.png" alt="">
+                <img src="./Images/Ellipse_3.png" alt="CSS3 Icon">
               </div>
               <span>CSS</span>
```

Comment:
Kindly check the image path, it is not displaying😐

![image](https://user-images.githubusercontent.com/73854999/200682736-05d4db80-7795-40d3-9acc-f2ad3a6be627.png)